assets/js/account/account-switch.js

```diff
@@ -0,0 +1,90 @@
+vcAccountSwitchController.$inject = ['multiAccountService', '$log'];
+function vcAccountSwitchController (multiAccountService, $log) {
+    var $ctrl = this;
+    $ctrl.currentAccount = multiAccountService.getCurrentAccount();
+
+    $ctrl.accounts = multiAccountService.getAccounts();
+    $log.log($ctrl.accounts);
```

Comment:
Do we really need any logging here? Or you just forgot to delete it?

Reply:
Fixed